lib/tests/modules/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib ? import <nixpkgs/lib>, modules ? [] }:
+{ lib ? import ../.., modules ? [] }:
```

Comment:
I think this is a legacy of when I created the test suite, to check against old tree of Nixpkgs, with `-I nixpkgs=...`.

Nothing to worry about today.